packages/react-core/src/components/ExpandableSection/__tests__/__snapshots__/ExpandableSection.test.tsx.snap

```diff
@@ -58,6 +58,7 @@ exports[`Disclosure ExpandableSection 1`] = `
   >
     <button
       aria-controls="content-id"
+      aria-expanded="false"
```

Comment:
The updates to these snapshots were needed because the last v5 PR didn't update them and they were failing.